README.md

```diff
@@ -973,7 +973,7 @@ Other Style Guides
 
     // good
     foo(() => {
-      bool = true;
+      return bool = true;
```

Comment:
altho in this case, your change is bad - we should never conflate assignment with expressions.